internal/overlay/overlay.go

```diff
@@ -67,6 +67,7 @@ func Apply(schema string, overlayFile string, w io.Writer) error {
 	}
 
 	enc := yaml.NewEncoder(w)
+	enc.SetIndent(2)
```

Comment:
This ensures we stop auto-formatting overlayed specs as single line files which start with { and end with }